README.md

```diff
@@ -214,6 +215,7 @@
 | Stable Diffusion | 2021.12.20 | CVPR 2022 | [High-Resolution Image Synthesis With Latent Diffusion Models](https://arxiv.org/abs/2112.10752) |
 | BLIP | 2022.01.28 | ICML 2022 | [BLIP: Bootstrapping Language-Image Pre-training for Unified Vision-Language Understanding and Generation](https://arxiv.org/abs/2201.12086) |
 | InstructGPT | 2022.03.04 | NeurIPS 2022 | [Training language models to follow instructions with human feedback](https://arxiv.org/abs/2203.02155) |
+| GPT-NeoX | 2022.04.14 | | (GPT-NeoX-20B: An Open-Source Autoregressive Language Model)[https://arxiv.org/pdf/2204.06745.pdf] |
```

Comment:
This paper seems to have been accepted at the [BigScience Research Workshop](https://bigscience.huggingface.co/acl-2022). I would like to reveal this information to show respect to the author. 

```suggestion
| GPT-NeoX | 2022.04.14 | BigScience Research Workshop at ACL 2022 | [GPT-NeoX-20B: An Open-Source Autoregressive Language Model](https://aclanthology.org/2022.bigscience-1.9/) |
```

Reply:
Nice catch!